### PR TITLE
chore: migrating cropper-modal to ES6

### PR DESCRIPTION
**Pull request by maze-runnar**
Fix for #3618 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
@kushthedude please review

File: app/components/modals/cropper-modal.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import ModalBase from 'open-event-frontend/components/modals/modal-base';
-
-export default ModalBase.extend({
+import { action } from '@ember/object';
+export default class extends ModalBase { 
   onVisible() {
     let viewport = {};
-    let factor = 150;
+    let factor = 750;
```
Comment (kushthedude) [inline app/components/modals/cropper-modal.js]: Revert

File: app/components/modals/cropper-modal.js [hunk 2]
```diff
@@ -16,27 +16,28 @@ export default ModalBase.extend({
...
+  
+  @action
+  resetImage() {
+    this.onHide();
+    this.onVisible();
+  }
+  @action
+  cropImage() {
+    this.$('img').croppie('result', 'base64', 'original', 'jpeg', 1).then(result => {
+      if (this.onImageCrop) {
```
Comment (kushthedude) [inline app/components/modals/cropper-modal.js]: Remove the 1 from arguement

**Comment by kushthedude** [comment]
Fix build

**Comment by kushthedude** [comment]
> Fix build

.

**Comment by maze-runnar** [comment]
@kushthedude please review and merge .

**Comment by uds5501** [comment]
@kushthedude update the branch again please and see if it's good to merge after that

**Comment by kushthedude** [comment]
> @kushthedude update the branch again please and see if it's good to merge after that

Its on hold because of the Croppie PR. ONCE the Croppie one is merged here will be conflicts hence putted on hold.

**Comment by kushthedude** [comment]
@maze-runnar Resolve conflicts

File: app/components/modals/cropper-modal.js [hunk 3]
```diff
@@ -16,27 +16,28 @@ export default ModalBase.extend({
...
     }
-  },
 
-  actions: {
-    resetImage() {
-      this.onHide();
-      this.onVisible();
-    },
-    cropImage() {
-      this.$('img').croppie('result', { type: 'base64', size: 'original', quality: 1, format: 'jpeg' }).then(result => {
```
Comment (kushthedude) [inline app/components/modals/cropper-modal.js]: You removed the very change you made in the RECENT PR

File: app/components/modals/cropper-modal.js [hunk 4]
```diff
@@ -16,27 +16,28 @@ export default ModalBase.extend({
...
   }
-});
+  @action
+  resetImage() {
+    this.onHide();
+    this.onVisible();
+  }
+  @action
+  cropImage() {
+    this.$('img').croppie('result', 'base64', 'original', 'jpeg').then(result => {
```
Comment (kushthedude) [inline app/components/modals/cropper-modal.js]:
```suggestion
    this.$('img').croppie('result', { type: 'base64', size: 'original', quality: 1, format: 'jpeg' }).then(result => {
```

**Pull request merged**